src/db/index.js

```diff
@@ -3,11 +3,20 @@ const serviceAccount = require("../../firebase-credentials.json");
 
 // initialize firebase store
 admin.initializeApp({
-  credential: admin.credential.cert(serviceAccount)
+  credential: admin.credential.cert(serviceAccount),
+  databaseURL: "https://woke-company.firebaseio.com/"
 });
 
-const db = admin.firestore();
+const db = admin.database();
 
-module.exports = {
+// helpers
+const read = nameOrPath =>
```

Comment:
Made a helper here to read lists of data, you can reuse this for the other things you have to read from firebase